index.html

```diff
@@ -0,0 +1,248 @@
+                    <p class="speaker-desc">As President of the Ghana Esports Federation, Igor has coordinated many Esports events from the grass-root to the top of the ladder. He is known to have the biggest collection of top players in a single tournament.</p>
+                </div>
+            </div>
+            <div class="speaker-box">
+                <img src="assets/speakers/speaker_04.png" alt="speaker1-img" class="speaker-image">
+                <div class="speaker-desc-box">
+                    <h4 class="speaker-name">Igor</h4>
+                    <p class="speaker-title">Madagastar Esports Arena, West-Hills Mall, Weija-Accra</p>
+                    <p class="speaker-desc">As President of the Ghana Esports Federation, Igor has coordinated many Esports events from the grass-root to the top of the ladder. He is known to have the biggest collection of top players in a single tournament.</p>
+                </div>
+            </div>
+            <div class="speaker-box">
+                <img src="assets/speakers/speaker_05.png" alt="speaker1-img" class="speaker-image">
+                <div class="speaker-desc-box">
+                    <h4 class="speaker-name">Igor</h4>
+                    <p class="speaker-title">Madagastar Esports Arena, West-Hills Mall, Weija-Accra</p>
+                    <p class="speaker-desc">As President of the Ghana Esports Federation, Igor has coordinated many Esports events from the grass-root to the top of the ladder. He is known to have the biggest collection of top players in a single tournament.</p>
+                </div>
+            </div>
+            <div class="speaker-box">
+                <img src="assets/speakers/speaker_06.png" alt="speaker1-img" class="speaker-image">
+                <div class="speaker-desc-box">
+                    <h4 class="speaker-name">Igor</h4>
+                    <p class="speaker-title">Madagastar Esports Arena, West-Hills Mall, Weija-Accra</p>
+                    <p class="speaker-desc">As President of the Ghana Esports Federation, Igor has coordinated many Esports events from the grass-root to the top of the ladder. He is known to have the biggest collection of top players in a single tournament.</p>
+                </div>
+            </div> -->
+        </div>
+        <div class="show-btn">
+            <button id="more" class="show-more">
+                <span>MORE</span>
+                <img class="drop-down" src="assets/arrow-down.png" alt="drop-down-arrow">
+            </button>
+            <button id="less" class="show-more hide-speaker">
+                <span>LESS</span>
+                <img class="drop-down" src="assets/arrow-up.png" alt="drop-down-arrow">
+            </button>
+        </div>
+    </section>
+
+<!-- PARTNERS SECTION ----------------------------------------------------------------------------- -->
+<section class="partners hide-class">
```

Comment:
- [ ] **Mobile size: HOME**
- After clicking on more, I noticed that you don't have a Partner section, Please add the partner section
- 
![Capture](https://user-images.githubusercontent.com/63972670/212201957-1f32e5da-3482-456e-8af3-e7cb14842b9b.PNG)

Reply:
Hi @OLIPLICHE ,
The partner section of this project, according to the original design is found at the about page so I placed it in that page. Kindly verify from `about.html` where the partners section is present.